apps/researcher/tailwind.config.ts

```diff
@@ -0,0 +1,7 @@
+import sharedConfig from 'tailwind-config/tailwind.config';
+
+const config = {
```

Comment:
Would it be possible to add `export default` to this line, so as to make immediately clear that it's being exported as a default?